docs(openid-connect) update docs to match 1.5 release (#2118)
Kong/docs.konghq.com: app/_hub/kong-inc/openid-connect/index.md

(from requerst)   | `no`

#### config.preserve_query_args

> When using authorization code flow the original query parameters (along with

Reviewer: ```suggestion
When using authorization code flow, the original query parameters (along with
```

could initial cap Auth Code Flow

`3600`            | `no`

#### config.session_cookie_idletime

> Session cookie idletime in seconds. It holds the cookie idletime in seconds in the future.

Reviewer: ```suggestion
Session cookie idle time in seconds. It holds the cookie idle time in seconds in the future.
```

be safely turned off. Please always check Kong error logs, as the plugins
in this suite usually give very clear information of what went wrong.

The default value was changed from `true` to `false` with plugin release

> `1.5.0` as it was causing more problems than it was worth.

Reviewer: ```suggestion
`1.5.0` because it was causing problems rather than solving them.
```
